Screenshot-OCR.xcodeproj/project.pbxproj

```diff
@@ -488,7 +488,7 @@
 				CODE_SIGN_IDENTITY = "Mac Developer";
 				CODE_SIGN_STYLE = Automatic;
 				COMBINE_HIDPI_IMAGES = YES;
-				DEVELOPMENT_TEAM = AGPAHW954E;
```

Comment:
You can remove this modification from your PR entirely with another commit that reverses this change.
```
git checkout origin/master --Screenshot-OCR.xcodeproj/project.pbxproj
git commit -m "Removed a modified file from pull request"
```